roofit/roofitcore/src/NormalizationHelpers.cxx

```diff
@@ -114,6 +114,8 @@ std::vector<std::unique_ptr<RooAbsArg>> unfoldIntegrals(RooAbsArg const &topNode
          if (currNormSet.empty())
             continue;
 
+         // TODO. Understand why this call to getVal() is necessary (without
+         // it, test 28 in stressRooFit is failing).
```

Comment:
It might be setting up some cached state, and a fallback (or not batch enabled thing) might call the old interface.

Reply:
Since you use `AClean`, the state will never get recomputed.